modules/hardware/polarfire/debug-packages.nix

```diff
@@ -0,0 +1,16 @@
+{pkgs, ... } :
+{
+  environment.systemPackages = with pkgs; [ 
```

Comment:
Most of these come from stdenv and can be used in `callPackage` with whatever nix declared development packages. We should not put them to host and to `systemPackages` - who ever needs these can use specific `nix-shell -p <package>`or `nix develop` with flakes. Leaving them out will also reduce the image size and decrease build time.